router/router.go

```diff
@@ -362,9 +363,8 @@ func (worker *workerT) workerProcess() {
 			userID := job.UserID
 
 			var parameters JobParametersT
-			err := json.Unmarshal(job.Parameters, &parameters)
-			if err != nil {
-				worker.rt.logger.Error("Unmarshal of job parameters failed. ", string(job.Parameters))
+			if err := json.Unmarshal(job.Parameters, &parameters); err != nil {
+				panic(fmt.Errorf("unmarshalling of job parameters failed (%s): %w", string(job.Parameters), err))
```

Comment:
Instead of panicking, let's log relevant information, raise alert and abort the job?

Reply:
Failing to unmarshal bytes stored in a jsonb column is highly unlikely, practically impossible. Thus I don't think we should care too much about it, to let it complicate router's logic even further.
It is just that logging an error instead of panicking in such a fundamental issue seemed to be weird in the first place.